examples/basics/quick_start.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>I think you can remove the comment "specify the media type" since it doesn't add much.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>